examples/stable/postgresql-ha/hook-blueprint/postgres-ha-hook.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: postgresql-hooks
+actions:
+  postRestoreHook:
+    phases:
+    - func: Wait
+      name: WaitForPodReady
+      args:
+       timeout: 240s
+       conditions:
+          anyOf:
+          - condition: '{{ if { $.status.containerStatuses[].ready } }}true{{ else }}false{{ end }}'
+            objectReference:
+              apiVersion: v1
+              resource: pods
+              name: '{{ .StatefulSet.Name }}-0'
+              namespace: '{{ .StatefulSet.Namespace }}'
+          - condition: '{{ if { $.status.containerStatuses[].ready } }}true{{ else }}false{{ end }}'
+            objectReference:
+              apiVersion: v1
+              resource: pods
+              name: '{{ .StatefulSet.Name }}-1' 
+              namespace: '{{ .StatefulSet.Namespace }}'
```

Comment:
I think this is redundant, for a statefulset, while scaling up, the 2nd pod won't get created unless 1st one is up and ready. Eventually, we are checking only 1st pod status only to decide the result as we are using `anyOf` clause

Reply:
makes sense, I will update it

Reply:
If we want to wait till both pods become ready, then we should use `allOf`

Reply:
No, we dont want all of them to be ready